app/actions/index.js

```diff
@@ -23,6 +25,10 @@ window.fetch = new Fetch({
   binaryContentTypes: ["image/", "video/", "audio/", "application/gzip"]
 }).build();
 
+const obsdb = createOsmp2p("obs");
+const osmdb = createOsmp2p("osm");
+const osm = osmp2p(obsdb, osmdb);
```

Comment:
Doesn't a DB pair get created in `app/screens/Observations/map.js` as well? If they're both open at the same time, you will get `LOCK`file errors.